### PR TITLE
Fedora32 installation

### PR DESCRIPTION
**Pull request by adrianomarto**
I changed the installation script so it works on Fedora32. It might work on previous versions as well, but I have only tested it on Fedora32.

### CODE REVIEW

File: util/install.sh [hunk 1]
```diff
@@ -592,9 +601,11 @@ function nox13 {
 
 # "Install" POX
 function pox {
-    echo "Installing POX into $BUILD_DIR/pox..."
-    cd $BUILD_DIR
-    git clone https://github.com/noxrepo/pox.git
+    if [ -d "/path/to/dir" ]; then
```
Comment (lantz) [inline util/install.sh:604]: This is definitely not right.

File: util/install.sh [hunk 2]
```diff
@@ -397,7 +405,8 @@ function ovs {
     echo "Installing Open vSwitch..."
 
     if [ "$DIST" = "Fedora" -o "$DIST" = "RedHatEnterpriseServer" ]; then
-        $install openvswitch openvswitch-controller
+        $install openvswitch
```
Comment (lantz) [inline util/install.sh:408]: Older RHEL needed this apparently?

File: mininet/node.py [hunk 3]
```diff
@@ -1466,7 +1466,8 @@ def __init__( self, name, **kwargs ):
     def isAvailable( cls ):
         return (which( 'ovs-controller' ) or
                 which( 'test-controller' ) or
-                which( 'ovs-testcontroller' ))
+                which( 'ovs-testcontroller' ) or
+                which( 'ovs-ofctl' ))
```
Comment (lantz) [inline mininet/node.py:1470]: This is not correct - `ovs-ofctl` is not a controller.

**Review by lantz — CHANGES_REQUESTED** [review]

**Comment by lantz** [comment]
Some of the changes are not correct.

Can you confirm that it works on older Fedora/RHEL as well?

File: util/install.sh (hunk 1 above)
Comment (adrianomarto) [reply util/install.sh]: Ops! Fixed.

File: util/install.sh (hunk 2 above)
Comment (adrianomarto) [reply util/install.sh:408]: Probably, but I am not sure. Giving this problem a proper solution may require more effort than I thought. The Fedora package generated directly from ovenvswitch sources has the controller, but the official package included in Fedora doesn't. I will see what can be done to include the controller in the Fedora package. Installing openvswitch from the source is the only way to go until that part is solved.
https://bugzilla.redhat.com/show_bug.cgi?id=1914605

Once that part is solved, I might create a Fedora package for mininet myself.

File: mininet/node.py (hunk 3 above)
Comment (adrianomarto) [reply mininet/node.py]: Now that I looked at it more carefully, I realised that my change does not make any sense.
By the way, what should not work without a proper ovs-testcontroller? I am using mininet so emulate a network on a continuous integration server based on Fedora with this incorrect hack, and it seems to work! At least the nodes are created and they communicate to each other. I am just curious.

**Pull request closed without merging**